LICENSE

```diff
@@ -19,3 +19,10 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+
+Built distributions of pyhdf also include:
+Libary | Licence
```

Comment:
```suggestion
Libary | License
```
I realize this is an alternate spelling, but we should be consistent with other uses in this file and the filename.